test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,11 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull(null, null, null));
+        assertTrue(Utils.isAnyNull("abc", 123, null));
+        assertFalse(Utils.isAnyNull("abc", 123));
```

Comment:
May be add more test cases for:
* A single null object
* Empty list
* Combinations of multiple null, non-null objects etc.